nni/compression/pytorch/speedup/jit_translate.py

```diff
@@ -363,44 +366,11 @@ def arg_trans_layout(ivalue: Union[int, torch.layout]):
     'layout': arg_trans_layout,
 }
 
-schema_fix_dict = {
-    # functinon 'to', 'randint', and 'sparse_coo_tensor' has different schema between python and c++.
-    # https://pytorch.org/docs/stable/jit_unsupported.html#ops-with-divergent-schemas-between-torch-python
-    """aten::to.device(Tensor(a) self, Device device, int dtype, bool non_blocking=False, bool copy=False, int? memory_format=None) -> (Ten
-    sor(a))""":
-        """aten::to.device(Tensor(a) self, Device device, int dtype, bool non_blocking=False, bool copy=False, *, int? memory_format=None)
-         -> (Tensor(a))""",
-    'aten::to.dtype(Tensor(a) self, int dtype, bool non_blocking=False, bool copy=False, int? memory_format=None) -> (Tensor(a))':
-        'aten::to.dtype(Tensor(a) self, int dtype, bool non_blocking=False, bool copy=False, *, int? memory_format=None) -> (Tensor(a))',
-    'aten::to.other(Tensor(a) self, Tensor other, bool non_blocking=False, bool copy=False, int? memory_format=None) -> (Tensor(a))':
-        'aten::to.other(Tensor(a) self, Tensor other, bool non_blocking=False, bool copy=False, *, int? memory_format=None) -> (Tensor(a))',
```

Comment:
because now i use a better way to do transform for special schemas. the old way cannot deal with to.dtype_layout and randint series.

the replacements of deleted part are in `table_fix_schema` now